design-proposals/network-binding-plugin/network-binding-plugin.md

```diff
@@ -270,6 +270,22 @@ The requirement to maintain certificates for the webhook could be mitigated usin
 
 For the alternative solutions please see [Appendix G](#appendix-g-alternatives-to-plugin-sidecar-container-resource-specification)
 
+##### Compute Container Resource Overhead
+
+For some plugins, an additional resource consumption can be expected from the virt-launcher pod compute container.
+For example, there could be need to execute an additional binary in the compute container.
+Since this binary has its own CPU and memory limits, they should be somehow accounted for.
+Another example could be increased resource consumption of the virt-stack resulting from using the plugin.
+
+Suggested solution:
+
+Shipping a mutating webhook as part of the plugin's deliverables.
```

Comment:
Same as for the sidecar resources (prev commit) reasoning, I would prefer the Kubevirt CR API extension option.

Reply:
Done.